detekt_custom.yml

```diff
@@ -816,6 +817,7 @@ datadog:
       - "kotlin.collections.List.findFirstForType(java.lang.Class)"
       - "kotlin.collections.List.firstOrNull()"
       - "kotlin.collections.List.firstOrNull(kotlin.Function1)"
+      - "kotlin.collections.List.firstNotNullOfOrNull(kotlin.Function1)"
```

Comment:
should come above according to the ordering

Reply:
removed due to the suggested change below